src/cc/bcc_elf.c

```diff
@@ -911,6 +928,32 @@ int bcc_elf_get_buildid(const char *path, char *buildid)
   return 0;
 }
 
+int bcc_elf_symbol_str(const char *path, size_t section_idx,
+                       size_t str_table_idx, char *out, size_t len)
+{
+  Elf *e;
+  int fd, err = 0;
+  const char *name;
+
+  if (!out || !len)
+    return -1;
+
+  if (openelf(path, &e, &fd) < 0)
```

Comment:
Can we have the `Module` just keep an opened ELF instance

Reply:
Yes, but I think that would make the bcc_elf API inconsistent. It seems like all the other functions take a path to the binary. Do you think that's a good enough reason to keep it slightly inefficient?